pb_chime5/scripts/kaldi_run_rttm_libri_css.py

```diff
@@ -0,0 +1,247 @@
+
+    session_id = None  # Default: All sessions from database_rttm
+    storage_dir: str = None
+
+    database_rttm: str = None
+    activity_rttm: str = database_rttm
+
+    # session_to_audio_paths: A file with a mapping from session ID (or file ID) used
+    #                   in rttm to the actual audio file.
+    # e.g. session_to_audio_paths = {
+    #          session_1: multichannel.wav,
+    #          session_1: [ch1.wav, ch2.wav, ...],
+    # }
+    # Supported are json and yaml.
+    session_to_audio_paths = None
+
+    job_id = 1
+    number_of_jobs = 1
+
+    assert storage_dir is not None, (storage_dir, 'overwrite the storage_dir from the command line')
+    assert database_rttm is not None, (database_rttm, 'overwrite the database_rttm from the command line')
+    assert activity_rttm is not None, (database_rttm, 'overwrite the activity_rttm from the command line')
+
+    if dlp_mpi.IS_MASTER:
+        experiment.observers.append(FileStorageObserver.create(str(
+            Path(storage_dir).expanduser().resolve() / 'sacred'
+        )))
+
+@experiment.named_config
+def my_test_rttm():
+    database_rttm = '/scratch/hpc-prf-nt1/cbj/net/vol/boeddeker/chime6/kaldi/egs/chime6/s5_track2_download/data/dev_beamformit_dereverb_stats_seg/rttm.U06'
+
+
+# get_enhancer = experiment.capture(get_enhancer)
+
+@experiment.capture
+def get_sessions(database_rttm):
+    database_rttm = Path(database_rttm)
+    database_rttm_text = database_rttm.read_text()
+    sessions: set = {line.split()[1] for line in database_rttm_text.splitlines()}
+    return sessions
+
+@experiment.capture
+def get_enhancer(
+    database_rttm,
+    activity_rttm,
+
+    # session_to_audio_paths: A file with a mapping from session ID (or file ID) used
+    #                   in rttm to the actual audio file.
+    # e.g. session_to_audio_paths = {
+    #          session_1: multichannel.wav,
+    #          session_1: [ch1.wav, ch2.wav, ...],
+    # }
+    # Supported are json and yaml.
+    session_to_audio_paths=None,
+
+    # chime6_dir='/net/fastdb/chime6/CHiME6',
+    # multiarray='outer_array_mics',
+    context_samples=240000,
+
+    wpe=True,
+    wpe_tabs=10,
+    wpe_delay=2,
+    wpe_iterations=3,
+    wpe_psd_context=0,
+
+    activity_garbage_class=True,
+
+    stft_size=1024,
+    stft_shift=256,
+    stft_fading=True,
+
+    bss_iterations=20,
+    bss_iterations_post=1,
+
+    bf_drop_context=True,
+
+    bf='mvdrSouden_ban',
+    postfilter=None,
+):
+    from pb_chime5.core_chime6_rttm import (
+        Enhancer,
+        get_database,
+        WPE,
+        Activity,
+        GSS,
+        Beamformer,
+        RTTMDatabase,
+    )
+    if session_to_audio_paths is None:
+        from paderbox.array import intervall as array_intervall
+
+        sessions: set = get_sessions(database_rttm)
+        assert len(sessions) == 1, sessions
+
+        files = list(Path(database_rttm).parent.glob('*.wav'))
```

Comment:
I am not sure, what the best way is to obtain the observation.
The default is to assume, that the wav file next to the rttm file is the observation.
Feel free to propose an alternative (Do you know a common way that generalizes to 1 file per channel?).